lib/pages/image_viewer/image_viewer.dart

```diff
@@ -60,6 +75,28 @@ class ImageViewerController extends State<ImageViewer> {
     }
   }
 
+  void onDoubleTap() {
```

Comment:
How about this widget: https://api.flutter.dev/flutter/widgets/InteractiveViewer-class.html

Reply:
WDYM ? We already use interactiveViewer (image_viewer_view.dart)